src/Components/Profile.js

```diff
@@ -19,8 +19,8 @@ function Profile({ name, bio, avatar, total }) {
         </Avatar>
         <h1 className="p-m-2">{name}</h1>
       </div>
-      <div className="p-d-flex p-jc-center">
-        <p>{bio}</p>
+      <div className="p-d-flex p-jc-center w-50">
+        <p className="w-50" >{bio}</p>
```

Comment:
Please make a separate pull request for this. One for your profile addition and one for this. 🙂

Reply:
@amittri1025 try [this](https://stackoverflow.com/a/34027874/11991177).

This helped me too.

Reply:
@harshraj8843 kindly explain this process of adding different commit and creating several PR.

Reply:
@amittri1025 go to your local version, and create a new branch - `git checkout -b [nameOfNewBranch]`. Then push to that branch, by `git push origin [branchName]` .

Documentation, that may help :  https://gist.github.com/nanusdad/7e516743e5e709073f7e

Online branching : https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/managing-branches

Any more questions, feel free to ask. thanks.

Reply:
@amittri1025 In my opinion, close this pull request and open two new pull requests :-

    - for `amittri1025 added`
    - for `Added max-width to paragraph overflowing`

### for ***amittri1025 added***

```
git fetch --all
git checkout -b add-profile upstream/main
git cherry-pick 95a04fd318f05b7faf3c396cd76b7f09bd34675d
git push -u origin add-profile
```

Then open new pull request from branch `add-profile`

### for ***Added max-width to paragraph overflowing***

```
git fetch --all
git checkout -b max-width upstream/main
git cherry-pick 444f56f6c5bdbe92609b45cfd5ebfd202b061ec5
git push -u origin max-width
```

Then open new pull request from branch `max-width`